src/psyclone/domain/nemo/transformations/nemo_arrayrange2loop_trans.py

```diff
@@ -347,23 +363,15 @@ def validate(self, node, options=None):
                 if operator is BinaryOperation.Operator.UBOUND:
                     continue
 
-            # We allow any references that are part of a structure syntax
+            # We allow any references that are part of a structure syntax, we
```

Comment:
"...syntax - we analyse its child components..."

Reply:
Done